docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -80,16 +80,14 @@ cd mongodb-panache-quickstart
 This command generates a Maven structure importing the RESTEasy/JAX-RS, JSON-B and MongoDB with Panache extensions.
 After this, the `quarkus-mongodb-panache` extension has been added to your `pom.xml`.
 
-If you don't want to generate a new project you can add the dependency in your `pom.xml`:
+If you don't want to generate a new project, add the dependency in your `pom.xml`:
```

Comment:
I personally like the previous version better

Reply:
Yeah, it's Emmanuel's guidance. He prefers when we drop the "you can" and so on. I wasn't convinced at first but it has grown on me, especially when you have a lot of instructions.